### ISSUE TITLE
Windos XP 32 bit

### ISSUE THREAD

**Issue by alexi3** [openhardwaremonitor/openhardwaremonitor#1498]
Hi,
on https://openhardwaremonitor.org/ is written: "Hardware Monitor software runs on 32-bit and 64-bit Microsoft Windows XP"
I am using Windows XP SP3 32 bit
OpenHardwareMonitor 0.7.1 and 0.8.0 is running.
When starting OpenHardwareMonitor.exe 0.9.6 I get "OpenHardwareMonitor.exe ist keine zulässige Win32 Anwendung"

**Comment by DarthDestroyer** [comment]
okay, i will not ask, why you use xp!

But i bet there is a .net framework change at 0.8...

Can you compile by yourself?

**Comment by alexi3** [comment]
Please have a look at Issue #1499
I have Microsoft Visual C# 2008 Express Edition.
I could successfully compile Open Hardware Monitor 0.1.19.0 (Feb. 12, 2010).
I can't compile current Open Hardware Monitor.
Open Hardware Monitor is good software and I want to help to improve it.

**Comment by DarthDestroyer** [comment]
you know, that you can get the actual community edition for free? Just an Information...

What error does it tell you, if you try to compile the last Version?

which .net version do you use with XP? Or which is installed?

**Comment by alexi3** [comment]
Yes I know but [https://visualstudio.microsoft.com/de/vs/community/]
shows 
VisualStudioSetup.exe is no vailid Win32-application (translated into english by me).

.net:
![Zwischenablage-1](https://user-images.githubusercontent.com/56302499/153730024-5d8a08dc-0340-42bc-8528-2f81c20a726c.png)

errors:
1st compile:
![Zwischenablage-2](https://user-images.githubusercontent.com/56302499/153730057-b286d412-f371-464f-879f-409c7bde3841.png)
2nd compile because of no success of 1st compile:
![Zwischenablage-3](https://user-images.githubusercontent.com/56302499/153730076-7a4eeb77-9415-40c3-afbe-c5d0274b0f34.png)

**Comment by alexi3** [comment]
Same Problem when starting OpenHardwareMonitor.exe 0.9.6 "OpenHardwareMonitor.exe ist keine zulässige Win32 Anwendung" after installing .NET 4.

**Comment by esiefker** [comment]
I can confirm this bug.  

It would be nice if it would be fixed, but otherwise removing XP from the list of supported systems and making a note on the webpage of the last version supported by XP would be helpful.  We probably don't actually need new features on hardware running XP.

**Comment by esiefker** [comment]
BTW, where can I download old versions of OHWM?
